exp/operator/config/default/manager_image_patch.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/openshift-gce-devel/operator-controller-amd64:dev
```

Comment:
Why is this coming from an openshift image repo? I would have expected something like:

```
gcr.io/k8s-staging-cluster-api/cluster-api-operator:operator-0.4
```

Reply:
I'll remove this, it came from my gce config